packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -1,13 +1,32 @@
-## Recent Activity
+## Recent account activity
+## All strings except title indicate an event that occurred from the user's account
+## These are displayed as a list with the date when the event occured
 
-recent-activity-title = Recent Account Activity
+recent-activity-title-v2 = Recent account activity
 
-recent-activity-account-create = Account was created
-recent-activity-account-disable = Account was disabled
-recent-activity-account-enable = Account was enabled
-recent-activity-account-login = Account initiated login
-recent-activity-account-reset = Account initiated password reset
-recent-activity-emails-clearBounces = Account cleared email bounces
+recent-activity-account-create-v2 = Account created
+recent-activity-account-disable-v2 = Account disabled
+recent-activity-account-enable-v2 = Account enabled
+recent-activity-account-login-v2 = Account login initiated
+recent-activity-account-reset-v2 = Password reset initiated
+recent-activity-emails-clearBounces-v2 = Email bounces cleared
+recent-activity-account-login-failure = Account login attempt failed
+recent-activity-account-two-factor-added = Two-step authentication enabled
+recent-activity-account-two-factor-requested = Two-step authentication requested
+recent-activity-account-two-factor-failure = Two-step authentication challenge failed
+recent-activity-account-two-factor-success = Two-step authentication challenge successful
+recent-activity-account-two-factor-removed = Two-step authentication removed
+recent-activity-account-password-reset-requested = Account requested password reset
+recent-activity-account-password-reset-success = Account password reset successful
+recent-activity-account-recovery-key-added = Account recovery key enabled
+recent-activity-account-recovery-key-challenge-failure = Account recovery key challenge failed
+recent-activity-account-recovery-key-challenge-success = Account recovery key challenge successful
```

Comment:
Could you add a comment to explain what a "challenge" is in this context?

Reply:
Oh, yes! Thanks you for the catch. I'd actually changed this in the fallback to replace "challenge" with verification. Also replaced "challenge" in lines 16-17 above. Please let me know if the changes are clearer, or if a comment would still be appreciated.

Reply:
Verification is clear enough, looks good now 👍